adapters/repos/db/aggregations_integration_test.go

```diff
@@ -78,37 +78,37 @@ func prepareCompanyTestSchemaAndData(repo *DB,
 
 		t.Run("creating the class", func(t *testing.T) {
 			require.Nil(t,
-				migrator.AddClass(context.Background(), kind.Thing, productClass))
+				migrator.AddClass(context.Background(), kind.Object, productClass))
```

Comment:
similar to above, having the kind still in the method signature isn't required anymore, but I don't think we need to do the cleanup right now

Reply:
I have created an issue for that change: https://github.com/semi-technologies/weaviate/issues/1366

Reply:
perfect, thanks